src/aqua/installation-spell/src/aqua/spell.aqua

```diff
@@ -161,11 +161,11 @@ func download_service(s: Service, ipfs: Multiaddr) -> []Hash:
     for m <- s.modules:
         get_wasm <- get_from(HOST_PEER_ID, m.wasm, ipfs)
         if get_wasm.success:
-            on HOST_PEER_ID:
-                try:
+            try:
+                on HOST_PEER_ID:
```

Comment:
Does that need to be executed on host and not on worker?

Reply:
Yes, because `aqua-ipfs` is available only on host.